### ISSUE TITLE
[Bug] Balance tooltip on progress report is poorly worded

### ISSUE THREAD

**Issue by jlaflin** [CorsixTH/CorsixTH#2579]
### Describe the issue

The Tooltip that's displayed on the Progress Report dialog for balance shows a negative win condition balance if the hospital balance (minus loans) is negative 

### Steps to Reproduce

1. Spend enough money to take your hospital balance to < $5000
1. Take out a $5000 loan
1. Spend enough money to take your hospital balance to < $5000 again (so that your 'real balance' is negative)
1. Go to the 'Status' / 'Progress Report' screen (F7)
1. Hover over the tooltip for Balance, see "Your bank balance should not be less than -10000. Currently it's -3450"

### Expected Behaviour

Tooltip to show "Your bank balance should not be less than 10000. Currently it's -3450" i.e. without the '-' before the win condition balance

### Save Game

Save game attached to report: No

### CorsixTH Version

0.67

### Operating System

Windows 10

### Theme Hospital Version

GOG.com

### Additional Information

![image](https://github.com/CorsixTH/CorsixTH/assets/3956940/0aff802a-7642-406e-a23f-851c2a242ef7)

**Comment by lewri** [comment]
What level were you playing on this one?

**Comment by jlaflin** [comment]
Level 2 'Sleepy Hollow', but it's also reproducible on other levels.

Attached save file with simple repro

[progress-negative.zip](https://github.com/user-attachments/files/15959014/progress-negative.zip)

**Comment by lewri** [comment]
On doing some reading... this appears related to #232  . This is fixed with a patch included for the next version to help improve behaviours in the progress report like this.

@tobylane can you confirm that your PR will also fix this behaviour? From my understanding, the progress report incorrectly included loans for the balance.

**Comment by tobylane** [comment]
On this level, the win goal is 10k, the lose goal is -10k. When the column is red it's using the lose value, that part is intentional. But the wording is wrong, or ambiguous. The line without the negative is the advice about winning, the line with the negative is the advice about losing. My next PR in this area currently maintains this, but gives us an opportunity to fix it.

**Comment by lewri** [comment]
As far as I can tell it's behaving as intended then, there's just ambiguity you have mentioned?

**Comment by tobylane** [comment]
Yes. it could look like this, a different message for the win for balance tooltip.
<img width="354" alt="Screenshot 2024-06-25 at 13 07 54" src="https://github.com/CorsixTH/CorsixTH/assets/552285/22870cd6-3aeb-4766-bc30-1fa89d64f3cc">

**Comment by jlaflin** [comment]
> On this level, the win goal is 10k, the lose goal is -10k. When the column is red it's using the lose value, that part is intentional. But the wording is wrong, or ambiguous. The line without the negative is the advice about winning, the line with the negative is the advice about losing. My ~~next~~ current PR in this area currently maintains this, but gives us an opportunity to fix it.

This makes sense, thanks! 

Maybe a simple fix for the ambiguity would be to add something like "(or you will lose the level)" to the end of the tooltip if the bar is red / the tooltip is showing the lose condition

When does the game decide that you've lost? That might be useful information to include in there too if it's decided on e.g. Jan 1st
